cfme/tests/cli/test_appliance_console.py

```diff
@@ -1382,7 +1381,13 @@ def test_appliance_console_key_fetch_negative():
             5.
             6. Check Encryption Key fetch failure.
     """
-    pass
+    appliance = temp_appliance_preconfig_funcscope
+    invalid_ip = "192.168.10.1"
+    command_set = ("ap", RETURN, "14", "Y", "2", invalid_ip, "root", "smartvm",
```

Comment:
We should pull the user and passwd from the yamls. Checkout `conf.credentials['default']`